### ISSUE TITLE
Broken requirement on cryptography

### ISSUE THREAD

**Issue by weddige** [nabla-c0d3/sslyze#424]
While the `Pipfile` was updated during #400, the `setup.py` remained the same:

https://github.com/nabla-c0d3/sslyze/blob/3.0.0/setup.py#L93
```python
    # Dependencies
    install_requires=[
        "nassl>=3.0.0,<3.2.0",
        "cryptography==2.5",
        "tls-parser>=1.2.0,<1.3.0",
        "typing_extensions ; python_version<'3.8'",  # To remove when we drop support for Python 3.7
    ],
```

`cryptography==2.5` has to be changed to `cryptography==2.8` to match the `Pipfile`.

**Comment by nabla-c0d3** [comment]
Fixed in https://github.com/nabla-c0d3/sslyze/commit/c15e69e864aee43ade7cfe11af93fa717af0d7e7 - Sorry I didn't see you had opened a pull request. Thanks!

**Comment by nabla-c0d3** [comment]
Fixed in 3.0.1.
